gossip/src/cluster_info.rs

```diff
@@ -1646,7 +1656,7 @@ impl ClusterInfo {
             .add_relaxed(num_nodes as u64);
         if self.require_stake_for_gossip(stakes) {
             push_messages.retain(|_, data| {
-                retain_staked(data, stakes);
+                retain_staked(data, stakes, false);
```

Comment:
for someone reading this code later, it is not clear what each of these `false/true` means.
I think it would be good to add some inline comments as in: 
```rust
retain_staked(data, stakes, /*drop_unstaked_node_instance:*/ false);
```